main.py

```diff
@@ -12,19 +12,19 @@
         with open('./saved/embeddings/rahul.pickle', 'rb') as f:
             embeddings = pickle.load(f)
     else:
-        miner = GoogleDataMiner(user='rahul', data_path='.')
-        google_embeddings = miner.mine_data()
+        fbminer = FbInstaDataMiner(user='rahul', data_path='.')
+        fb_embeddings = fbminer.mine_data()
 
         # Save embeddings
         with open('./saved/embeddings/rahul.pickle', 'wb') as f:
-            pickle.dump(google_embeddings, f)
+            pickle.dump(fb_embeddings, f)
 
-        fbminer = FbInstaDataMiner(user='rahul', data_path='.')
-        fb_embeddings = fbminer.mine_data()
+        miner = GoogleDataMiner(user='rahul', data_path='.')
+        google_embeddings = miner.mine_data()
 
         # Save embeddings
         with open('./saved/embeddings/rahul.pickle', 'wb') as f:
```

Comment:
This overwrites the file again. Do the merging first.

Reply:
Not sure what you mean? This is the same as the master branch's main.py, I just reversed the blocks. FBDataMiner writes to fb_embeddings and dumps to <user>.pickle, then Google Data Miner writes to google_embeddings and dumps to <user>.pickle, then merge.

Reply:
Fixed